### DISCUSSION TITLE
A question related to windows platform

### DISCUSSION THREAD

**Discussion by 1eedaegon** [tokio-rs/tokio#7027] [General] [answered]
I'm a Rust user with just a few months of experience.
I'm implementing AIO with Tokio and found out that it uses IOCP on the Windows platform.
Have any plans to support RIO in the future?

**Comment by Darksonn** [comment, marked as answer]
There are no such plans. There isn't active development on windows since we don't have any windows experts available.

For further questions about this, please ask on the [mio](https://github.com/tokio-rs/mio) repository instead, since that's where the relevant code lives.

**Reply by 1eedaegon** [reply]
I found the issue with the [link](https://github.com/tokio-rs/mio/issues/155)
 Thank you :)
